appmail/templates/admin/appmail/loggedmessage/template_name_filter.html

```diff
@@ -0,0 +1,32 @@
+{% load i18n %}
+
+{% comment %}
+Overrides django/contrib/admin/templates/admin/filter.html
```

Comment:
Might be worth noting the Django version you took this from to make it easier to compare with upstream & update in the future

Reply:
Yep, will do.